vital-religion/introduction.adoc

```diff
@@ -1,30 +1,29 @@
 == Introduction.
 
-When we look around us,
+When we look around us
 and observe how many conflicting creeds and systems of
-religion are sedulously propagated in the present day,
+Christianity are continually promoted in the present day,
 our reflections on the subject should lead us to
-appreciate duly the privileges with which,
-through divine love and mercy, we are favored;
-the Light of Christ being given to illuminate our minds,
-and the Holy Scriptures being placed within our reach,
+properly appreciate the privileges with which,
+through divine love and mercy, we are favored.
+The Light of Christ has been given to illuminate our minds,
+and the holy scriptures have been placed within our reach,
 to which we can refer for instruction.
-In the Sacred Record, the doctrine preached by our Lord and Savior Jesus Christ,
-on the subject of the Redemption and Salvation of Man, is clearly exhibited.
-As no other subject of equal interest and importance can be presented to us,
-may these few pages,
-designed to point out from this high authority the way by
-which all mankind may come to the knowledge of the Truth,
-and be saved, prove the means, under the divine blessing,
+In that sacred record the doctrine preached by our Lord and savior Jesus Christ
+on the subject of the redemption and salvation of man is clearly exhibited.
+As there is no other subject of equal interest and importance,
+these few pages were written to point out from this high authority the way by which
+all mankind may come to the knowledge of the truth and be saved.
+May they prove the means, under the divine blessing,
 not only of convincing the merely nominal Christian of the imminent danger he is in;
-but also of encouraging the weak yet sincere believer,
-to hold fast the "`grain`" of living faith; (Matt. 17:20) that so,
+but also of encouraging the weak yet sincere believer
+to hold fast the "`grain`" of living faith; (Matt. 17:20) so that,
 amid all the conflicts with which he may be proved,
-his confidence may be steadily fixed on Him, who is the "`Wisdom of God,
-and the Power of God.`" (1 Cor. 1:24)
+his confidence may be steadily fixed on Him who is the "`wisdom of God,
+and the power of God.`" (1 Cor. 1:24)
```

Comment:
I agree with you in taking out a bunch of these commas, and random capitalizations.